exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/SequencesTests.kt

```diff
@@ -61,6 +62,39 @@ class SequencesTests : DatabaseTestsBase() {
         }
     }
 
+    @Test
+    fun `testInsertWithCustomSequence`() {
+        val tester = object : Table("tester") {
+            val id = integer("id").autoIncrement(myseq)
```

Comment:
I would make a separate sequence instance for the test to make it easier to discover and debugging